std/machines/memory_bb.asm

```diff
@@ -0,0 +1,76 @@
+use std::array;
+use std::machines::range::Byte2;
+
+// A read/write memory, similar to that of Polygon:
+// https://github.com/0xPolygonHermez/zkevm-proverjs/blob/main/pil/mem.pil
+machine Memory(byte2: Byte2) with
+    latch: LATCH,
+    operation_id: m_is_write,
+    call_selectors: selectors,
+{
+    // lower bound degree is 65536
+
+    operation mload<0> m_addr, m_step -> m_value1, m_value2;
+    operation mstore<1> m_addr, m_step, m_value1, m_value2 ->;
+
+    let LATCH = 1;
```

Comment:
I'm confused at this point. Is it 1 or [1]*? Is it just a one row computation?

Reply:
I think it translates to a fixed column so same as [1]*